lib/posix/clock.c

```diff
@@ -9,6 +9,7 @@
 #include <zephyr/posix/sys/time.h>
 #include <zephyr/internal/syscall_handler.h>
 #include <zephyr/spinlock.h>
+#include <zephyr/posix/unistd.h>
```

Comment:
minor nit as well: this could be inserted after `sys/time.h` to be in lexographic order, but not blocking.

Reply:
Done